commons-db/src/main/java/com/palantir/nexus/db/pool/RetriableTransactions.java

```diff
@@ -171,6 +171,10 @@ public Throwable getError() {
                                     now,
                                     e);
                         }
+                        if (cm.isClosed()) {
+                            log.warn("Aborting tx retry, underlying connection is closed", e);
```

Comment:
Nit as message should likely be an error and clarify the connection manager itself is closed

```suggestion
                            log.error("Aborting transaction retry, underlying connection manager is closed", e);
```